src/main/java/org/junitpioneer/jupiter/package-info.java

```diff
@@ -3,18 +3,23 @@
  *
  * <p>Check out the following types for details:
  * <ul>
- *     <li>{@link org.junitpioneer.jupiter.CartesianProductTest}</li>
- *     <li>{@link org.junitpioneer.jupiter.ClearSystemProperty} and {@link org.junitpioneer.jupiter.SetSystemProperty}</li>
  *     <li>{@link org.junitpioneer.jupiter.ClearEnvironmentVariable} and {@link org.junitpioneer.jupiter.SetEnvironmentVariable}</li>
+ *     <li>{@link org.junitpioneer.jupiter.ClearSystemProperty} and {@link org.junitpioneer.jupiter.SetSystemProperty}</li>
  *     <li>{@link org.junitpioneer.jupiter.DefaultLocale} and {@link org.junitpioneer.jupiter.DefaultTimeZone}</li>
  *     <li>{@link org.junitpioneer.jupiter.DisabledUntil}</li>
  *     <li>{@link org.junitpioneer.jupiter.ReportEntry}</li>
  *     <li>{@link org.junitpioneer.jupiter.RetryingTest}</li>
- *     <li>{@link org.junitpioneer.jupiter.StdIoExtension} and {@link org.junitpioneer.jupiter.StdIo}</li>
+ *     <li>{@link org.junitpioneer.jupiter.StdIo}</li>
  *     <li>{@link org.junitpioneer.jupiter.Stopwatch}</li>
  * </ul>
  *
- * Also check the {@link org.junitpioneer.jupiter.params params} package.
+ * Also check the following subpackages:
```

Comment:
Since there are no "subpackages", I would suggest:

```suggestion
 * Also check the following packages:
```